src/cascadia/WindowsTerminal/WindowEmperor.cpp

```diff
@@ -98,20 +94,8 @@ bool WindowEmperor::HandleCommandlineArgs()
         }
     }
 
-    // Get the requested initial state of the window from our startup info. For
-    // something like `start /min`, this will set the wShowWindow member to
-    // SW_SHOWMINIMIZED. We'll need to make sure is bubbled all the way through,
-    // so we can open a new window with the same state.
-    STARTUPINFOW si;
-    GetStartupInfoW(&si);
-    const uint32_t showWindow = WI_IsFlagSet(si.dwFlags, STARTF_USESHOWWINDOW) ? si.wShowWindow : SW_SHOW;

```

Comment:
FYI There's one slight difference between this implementation and the way `nCmdShow` is generated. `__scrt_get_show_window_mode`'s default value is `SW_SHOWDEFAULT`. I believe in our case that difference doesn't really matter though.